providers/vertica/src/airflow/providers/vertica/hooks/vertica.py

```diff
@@ -67,7 +67,7 @@ class VerticaHook(DbApiHook):
 
     def get_conn(self) -> connect:
         """Return vertica connection object."""
-        conn = self.get_connection(self.vertica_conn_id)  # type: ignore
+        conn = self.get_connection(self.vertica_conn_id)
```

Comment:
@potiuk / @gopidesupavan  I am scratching my head and taking a look to the code.... as well as to the history. The field `vertica_conn_id` actually is not existing and seems invalid - for thi sis actually broken and can not be used. `type ignore` is just masking the problem currently.

I am not sure what is "right", I assume an `__init__()` is missing which defined the field (but this would be breaking for all users to define this in constructor) or it would need to use `conn_name_attr`.
```suggestion
        conn = self.get_connection(self.conn_name_attr)
```
```suggestion
        conn = self.get_connection(self.vertica_conn_id)
```

Reply:
yes

Reply:
conn_name_attr

Reply:
yeah looks like bug, we can set `default_conn_name` ?

Reply:
Changing it now to `self.get_connection(self.get_conn_id())` which makes most sense.

Reply:
its good that mypy exposed actual bugs , like this one and other providers google and samba